pkg/cli/service.go

```diff
@@ -367,12 +367,23 @@ func (service searchService) getCveByImage(ctx context.Context, config searchCon
 }
 
 func groupCVEsBySeverity(cveList []cve) []cve {
-	high := make([]cve, 0)
-	med := make([]cve, 0)
-	low := make([]cve, 0)
+	var (
+		unknown  = make([]cve, 0)
+		none     = make([]cve, 0)
+		high     = make([]cve, 0)
+		med      = make([]cve, 0)
+		low      = make([]cve, 0)
+		critical = make([]cve, 0)
+	)
 
 	for _, cve := range cveList {
 		switch cve.Severity {
```

Comment:
Still thinking about the fact we found this bug pretty late.
Do you think it would make sense to add a default, in case they add more severities we don't handle today? At least we would see the issues in the cli output one way or another.